src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -237,7 +237,11 @@
     <EmccCompile SourceFiles="@(_WasmSourceFileToCompile)" Arguments='"@$(_EmccDefaultFlagsRsp)" "@$(_EmccCompileRsp)"' EnvironmentVariables="@(EmscriptenEnvVars)" />
   </Target>
 
-  <Target Name="_WasmLinkDotNet">
+  <ItemGroup Condition="'$(Configuration)' == 'Debug'">
+    <_MonoComponent Include="hot_reload;diagnostics_tracing;debugger" />
```

Comment:
is diagnostics_tracing event source? if so we don't need ever (I think)

Reply:
I am not sure whether is is event source, probably is. It is now disabled for default runtime build, so I disabled it as well.